bgpd/bgp_evpn_vty.c

```diff
@@ -2619,10 +2619,8 @@ static void evpn_show_all_routes(struct vty *vty, struct bgp *bgp, int type,
 		prefix_rd2str((struct prefix_rd *)&rd_rn->p, rd_str,
 			      sizeof(rd_str));
 
-		if (json) {
+		if (json)
 			json_rd = json_object_new_object();
-			json_object_string_add(json_rd, "rd", rd_str);
```

Comment:
Chirag,

Changes looks fine. One minor comment: why are removing this rd string from the output ?

Reply:
@srimohans The rd is displayed inside the bgp_evpn_show_route_rd_header() with json options. The outer json object is redundant.

Reply:
Thanks for the explanation. Could you please upload the json output after the code modifications ?